server/src/main/java/org/elasticsearch/rest/action/admin/indices/RestGetFieldMappingAction.java

```diff
@@ -49,6 +58,12 @@ public RestChannelConsumer prepareRequest(final RestRequest request, final NodeC
         final String[] indices = Strings.splitStringByCommaToArray(request.param("index"));
         final String[] fields = Strings.splitStringByCommaToArray(request.param("fields"));
 
+        if (request.getRestApiVersion() == RestApiVersion.V_7 && request.hasParam(INCLUDE_TYPE_NAME_PARAMETER)) {
+            request.paramAsBoolean(INCLUDE_TYPE_NAME_PARAMETER, DEFAULT_INCLUDE_TYPE_NAME_POLICY);
```

Comment:
Why do we need to call `paramAsBoolean` here?

Reply:
good point, the value is ignored. Will fix to just use `param`